test/string/memmove.c

```diff
@@ -14,18 +15,92 @@
 #include <cmocka.h>
 // clang-format on
 
-static void memmove_test(void** state)
+struct test_memmove_data {
+	char template[64];
+	char buf[64];
+};
+
+static void fill_buffer(char* buf, size_t size, char start_value, char end_value)
+{
+	size_t i;
+	for (i=0; i< size; ++i)
+		buf[i] = (start_value + i) % (end_value - start_value + 1);
+}
+
+int setup_test(void **state)
 {
-	// TODO: "Memmove tests need to be implemented"
+	struct test_memmove_data* md = malloc(sizeof(struct test_memmove_data));
+	if (!md)
+		return -1;
```

Comment:
Please add braces around the body of the `if`

Reply:
Fixing, thanks.

Reply:
Hope all is well now!